### PR TITLE
Finesse readme

### PR DESCRIPTION
**Pull request by janet-can**
Additions to README for context. 
(FYI - File replaced because I originally made changes on the repo when it was private and ran into conflicts when tried to create a PR in a now public repo.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,66 @@
...
+  soda_scan:
+    runs-on: ubuntu-latest
+    name: Run Soda Scan
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.8.x"
```
Comment (gregkaczan) [inline README.md:34]: Setting up python is not required for the action itself.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,66 @@
...
+Where the scan results indicate an issue with data quality, Soda notifies you both in the PR comment, and by email so that you can investigate and address any issues before merging the PR into production.
+
+Refer to [Soda documentation](https://docs.soda.io/soda/quick-start-dev.html) for an example use case.
+
+## Use the Soda GitHub Action
+
+Add the action to your GitHub Workflow, as in the following example in the `Perform Soda Scan` step.
 
 ```yaml
-- name: Soda scan
```
Comment (gregkaczan) [inline README.md:10]: I would leave this snippet as simple example and put the full pipeline example below. In most cases i would assume, users already have a pipeline, they just need the actual action code only.

**Review by gregkaczan — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (janet-can) [reply README.md:10]: Sure, sounds good to me. Want to strip away the non-essential? Or shall I?

Comment (gregkaczan) [reply README.md:10]: Go for it, im off to sleep now :P

Comment (gregkaczan) [reply README.md:10]: I left it as-is actually, it is ok. Just removed the python part.

**Pull request merged**